kernel/kernel-default/src/main/java/io/delta/kernel/parquet/ParquetConverters.java

```diff
@@ -102,8 +87,61 @@ else if (typeFromClient instanceof FloatType) {
         else if (typeFromClient instanceof DoubleType) {
             return new DoubleColumnConverter(initialBatchSize);
         }
-        // else if (typeFromClient instanceof DecimalType) {
-        // }
+        else if (typeFromClient instanceof DecimalType) {
+            PrimitiveType primType = typeFromFile.asPrimitiveType();
```

Comment:
seems like this block can be in its own method.

Reply:
moved it to the separate file along with the converters